src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Layer5 and HashiCorp Launch Service Mesh Partnership"
+date:   2020-05-28 00:00:00 +08000
```

Comment:
In the `date` there is an extra 0 at the last.
It should be `date: 2020-05-28 00:00:00 +0800`